src/main/dialogs/captureToolDialog.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * Capture Tool Main Window Dialog
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+"use strict";
+
+var fs = require("fs"),
+    fluid = require("infusion"),
+    electron = require("electron"),
+    ipcMain = electron.ipcMain,
+    gpii = fluid.registerNamespace("gpii");
+
+require("../common/utils.js");
+
+require("./basic/dialog.js");
+require("./basic/blurrable.js");
+require("./basic/centeredDialog.js");
+require("./basic/resizable.js");
+require("./basic/offScreenHidable.js");
+
+require("../../../node_modules/gpii-universal/gpii/node_modules/solutionsRegistry/src/js/SolutionsUtils.js");
+
+fluid.defaults("gpii.app.captureToolUtils", {
+    gradeNames: ["fluid.component"],
+    events: {
+        onCollectDiagnostics: null
```

Comment:
Explain that this is a pseudoevent and give a clue where it is operated from. Preferably give this component a meaningful name such as "diagnosticsCollector"

Reply:
explained, and renamed the component grade